### PR TITLE
exposed props of the CustomHTML block to the style so it can be extended

### PR DESCRIPTION
**Pull request by aditudorache**
## Amsterdam styled components pull request

Before opening a pull request, please ensure:

- [ ] The default export from a file matches the file name
- [ ] You have been following the guidelines, written in the [README](../blob/master/docs/CONTRIBUTING.md#user-content-conventions-and-rules) file
- [ ] Your code has the necessary tests written
- [ ] You have updated the [CHANGELOG.md unreleased sections](../blob/master/CHANGELOG.md#muser-content-unreleased)

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: packages/asc-ui/src/components/CustomHTMLBlock/CustomHTMLBlock.stories.tsx [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+import CustomHTMLBlockStyle from './CustomHTMLBlockStyle'
+import CustomHTMLBlock from './CustomHTMLBlock'
 
 storiesOf('Atoms/CustomHTMLBlock', module)
   .addDecorator(storyFn => (
     <div style={{ padding: '40px 10px' }}>{storyFn()}</div>
   ))
   .add('default', () => (
-    <CustomHTMLBlock>
+    <CustomHTMLBlockStyle>
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/CustomHTMLBlock/CustomHTMLBlock.stories.tsx]: why not just use CustomHTMLBlock here?

File: packages/asc-ui/src/components/CustomHTMLBlock/CustomHTMLBlockStyle.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ const ElementStyleMapping = {
   li: ListItemStyleCSS,
 }
 
-export default styled.div`
+const CustomHTMLBlock = styled.div`
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/CustomHTMLBlock/CustomHTMLBlockStyle.ts]: Then rename this to CustomHTMLBlockStyle ;)

File: packages/asc-ui/src/components/CustomHTMLBlock/CustomHTMLBlock.stories.tsx (hunk 1 above)
Comment (aditudorache) [reply packages/asc-ui/src/components/CustomHTMLBlock/CustomHTMLBlock.stories.tsx]: this construction with children is not supported by the `CustomHTMLBlock`, just by `CustomHTMLBlockStyle`

File: packages/asc-ui/src/components/CustomHTMLBlock/CustomHTMLBlockStyle.ts (hunk 2 above)
Comment (aditudorache) [reply packages/asc-ui/src/components/CustomHTMLBlock/CustomHTMLBlockStyle.ts]: done!

**Comment by royderks** [comment]
If you want to do a canary release to test this locally you can run the command `yarn publish:canary` from your branch. It's important to do this before pushing to the master branch @aditudorache

**Pull request merged**